config/prod.json

```diff
@@ -0,0 +1,31 @@
+{
+  "language": "Solidity",
+  "sources": {
+    "contracts/Util.sol": {
+      "urls": ["contracts/Util.sol"]
```

Comment:
Need to add a ci.json file

Reply:
No, we killed `ci.json` because it can't be used in ci due to the linking.

Reply:
Ah yes